src/scene/webscene/index.jsx

```diff
@@ -101,7 +104,9 @@ class Webscene extends Component {
         );
         // assign only fist ground layer to avoid out-of-sync layer settings, e.g. visibility
         // as there is no way (yet) to add a group layer to the ground of a SceneView
-        groundLayer = filteredGroundLayers[0];
+        groundLayer = filteredGroundLayers.length
+          ? filteredGroundLayers[0]
+          : null;
```

Comment:
This ternary operator is not needed. `undefined` is better than `null` for an item that doesn't exist, imo.
```suggestion
        groundLayer = filteredGroundLayers[0];
```